functions/functions/verify/create-verification-service.js

```diff
@@ -0,0 +1,10 @@
+const { createVerificationService } = require(Runtime.getFunctions()['twilio/verify'].path);
```

Comment:
I'm torn on having the twilio service creations be automatic via script for several reasons.

pros:
- automation hell yeah,i LOVE this
- less room for human error

cons:
- More to maintain codewise as service setup, logic, config could change over time
- Twilio SE's should know how to create a service quickly
- Allows Twilio SE room to customize anything they need up front rather than digging through the console

This thought pattern is a bigger q about how automatic we want the platform to be used in general by Twilio SE's. Perhaps there could be an option for SE's: lightning setup versus set it up yoself?